### ISSUE TITLE
bundling failed: index.js: [BABEL] /Users/aashnishah/dev/<project>/index.js: Unknown option: .name.

### ISSUE THREAD

**Issue by aashnisshah** [goatandsheep/react-native-dotenv#61]
Hey folks!

Been trying to get `react-native-dotenv` working in my project but have been running into the following problem:
`bundling failed: index.js: [BABEL] /Users/aashnishah/dev/<project>/index.js: Unknown option: .name.`

babel.config.js:
```
module.exports = function (api) {
  api.cache(true);
  return {
    presets: [
      "babel-preset-expo",
      "module:metro-react-native-babel-preset",
      "module:react-native-dotenv",
      {
        moduleName: "@env",
        path: ".env",
        blacklist: null,
        whitelist: null,
        safe: false,
        allowUndefined: true,
      },
    ],
  };
};
```

package.json: (tried it in both dependencies and devDependencies)
```
dependencies: {
    ...
    "react-native-dotenv": "^2.3.0",
    ...
}
```

in my code:
```
import {
  KEY
} from "@env";
...
<TEXT>{KEY}</TEXT>
```

**Comment by aashnisshah** [comment]
Fixed!

tl;dr: Follow the steps in the [migration docs](https://github.com/goatandsheep/react-native-dotenv/wiki/Migration-Guide) or _actually_ read the setup instructions.

I'd followed setup instructions elsewhere which followed the v1 setup. I missed the part where we needed to move `"module:react-native-dotenv",` from the `presets` to the `plugins` section in the `babel.config.js` file

**Comment by adrach** [comment]
**I missed the part where we needed to move "module:react-native-dotenv", from the presets to the plugins section in the babel.config.js file**
this ⬆️ 
thanks!
